examples/notebooks/tests/test_notebooks_e2e.py

```diff
@@ -128,12 +135,15 @@ def _run_notebook(notebook_path: Path, expected_packages: list[str], timeout_sec
     [
         pytest.param("1_getting_started_with_nat.ipynb", ["getting_started"], 120, id="1_getting_started_with_nat"),
         pytest.param("3_adding_tools_to_agents.ipynb", ["retail_sales_agent"], 300, id="3_adding_tools_to_agents"),
-        pytest.param("4_multi_agent_orchestration.ipynb", ["retail_sales_agent"], 120,
-                     id="4_multi_agent_orchestration"),
-        pytest.param("5_observability_evaluation_and_profiling.ipynb", ["retail_sales_agent"],
+        pytest.param("4_mcp_setup_and_integration.ipynb", ["mcp_dev_workflow", "nat_simple_calculator"],
```

Comment:
Creating a new workflow (`nat_simple_calculator`), of the same name as the existing workflow is problematic, as this effectively uninstalls  `examples/getting_started/simple_calculator` replacing it with `examples/notebooks/simple_calculator` which is then removed at the end of the test. 

The problem is there are other tests which test `simple_calculator` or depend on tools that are in that package which would now fail.

I think the `install_example_code` should be updated to change the name of the example being installed.